sys-utils/nsenter.c

```diff
@@ -128,32 +130,29 @@ static int env_fd = -1;
 static int uid_gid_fd = -1;
 static int cgroup_procs_fd = -1;
 
-static void set_parent_user_ns_fd(void)
+static inline struct namespace_file *__next_nsfile(struct namespace_file *n, int namespaces, bool enabled)
```

Comment:
names containing double underscores are reserved for the C implementation, better call it `next_nsfile_` or `next_nsfile_f`